dataform/ecom-report-transform/README.md

```diff
@@ -0,0 +1,82 @@
+# `optimics/dataform-ecom-report-transform`
+
+# Ecom Report Transform
+
+## Overview
+
+This Dataform package is designed to help e-shops evaluate ecommerce data from
+Google Analaytics 4. It extracts information about ecommerce events and item parameters
+and stores it in a new dataset in BigQuery. The name of the dataset will be `ga4_datamart`.
+
+## Setting Up Dataform Repository
```

Comment:
This part is completely duplicated from [dataform-utm-source-filter](https://github.com/optimics/analytics/blob/master/dataform/utm-source-filter/README.md#creating-a-development-workspace). Could you extract it into a README.md on upper level to avoid duplications?